website/index.html

```diff
@@ -16,6 +16,12 @@ <h1 class="font-4xl md:font-hero font-thin md:mt-10 mb-8">{{ self.page_title() }
     <div class="tracking-wider">
       {{ download_thunderbird(force_direct=true, alt_copy=_('Free Download')) }}
     </div>
+    <div class="flex flex-wrap items-center justify-center pt-6">
+      <p class="font-md p-3 m-0">
+        {{ _('<span class="font-semibold">Want to try new features early?</span> Get the beta and be a part of the future of Thunderbird!') }}
+      </p>
+      <a href="https://support.mozilla.org/kb/thunderbird-beta" class="btn-beta rounded-full pt-2 pb-2 pr-3 pl-3 mb-0">{{ _('Get the Beta') }}</a>
```

Comment:
I don't think we want to link the support article here.
@wsmwk Should this link scroll the page to the bottom or directly trigger the beta download + donation page?

Reply:
@Alecaddd @ryanleesipes and I agreed to send this to the support page. This is my strong opinion that users be presented with information prior to installing beta, because users are putting important data at risk, not equivalent to Firefox beta.

Maybe at some point in the future when beta is more reliably stable, etc, we could reconsider this. 

Given that we took a different approach to implementing issue #302  the reference at the bottom should also be changed to point to the support page.

Triggering donations flow is going beyond the scope of this project IMO.

Reply:
Sounds good, thanks for the explanation.